sklearn/random_projection.py

```diff
@@ -82,7 +82,7 @@ def johnson_lindenstrauss_min_dim(n_samples, *, eps=0.1):
         Number of samples. If an array is given, it will compute
         a safe number of components array-wise.
 
-    eps : float or numpy array of float in ]0,1[, optional (default=0.1)
+    eps : float or numpy array of float in ]0,1[, default = 0.1
```

Comment:
Nitpick: You are not consistent in your whitespace usage (`default=0` and `default = 0`).

Reply:
Thanks for the feedback. I made the necessary changes now.